plugins/modules/packaging/language/pipx_info.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021, Alexei Znamensky <russoz@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: pipx_info
+short_description: Rretrieves information about applications installed with pipx
+version_added: 5.6.0
+description:
+    - Retrieve details about Python applications installed in isolated virtualenvs using pipx.
+options:
+    name:
+        description:
+            - Name of an application with C(pipx).
```

Comment:
Is `installed` missing here?
```suggestion
            - Name of an application installed with C(pipx).
```